src/components/Web3Network/index.tsx

```diff
@@ -1,5 +1,5 @@
 import { ChainId, CurrencyAmount } from '@kyberswap/ks-sdk-core'
-import React, { useMemo } from 'react'
+import { useMemo } from 'react'
```

Comment:
Why remove React. Is it work?

Reply:
we dont need this since react 17 was released

Reply:
I dont remember React17 allow this

Reply:
With the new transform, you can use JSX without importing React.
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

Reply:
cool